app/src/components/staking/DelegationsOverview.vue

```diff
@@ -52,17 +59,27 @@ export default {
   },
   data: () => ({
     delegations: [],
+    undelegations: [],
+    balances: [],
     delegationsLoaded: false,
+    undelegationsLoaded: false,
   }),
   computed: {
     ...mapState([`session`]),
     ...mapGetters([`address`, `currentNetwork`]),
     stakedBalance() {
-      const stakeBalance = this.balances.find(({denom}) => denom === this.currentNetwork.stakingDenom)
-      const liquidBalance =
-        Number(stakeBalance.total) - Number(stakeBalance.available)
+      const stakingDenomBalance = this.balances.find(
+        ({ denom }) => denom === this.currentNetwork.stakingDenom
```

Comment:
yes, thank you! i forgot to do this. also, it would be so nice if the balance query would accept denom as an argument and just return the balance of the thing we actually want. just an idea.

Reply:
the old balance query did this